content/v2.0/reference/glossary.md

```diff
@@ -1075,20 +1075,41 @@ This protocol is used when speed is desirable and error correction is not necess
 
 An implicit block that encompasses all Flux source text in a universe block.
 
+### unix timestamp
+
+The running total of a given unit of time since the **Unix Epoch (1970-01-01T00:00:00Z UTC)**.
+Express unix timestamps using different units of time (also known as **precision**).
```

Comment:
I think this may lose some of the subtlety of what a unix timestamp is. "Time" can be represented in many different ways/formats. The important thing to communicate here is that it is a counter for a specific unit of time. What about:

> The number of a given unit of time since the **Unix Epoch (1970-01-01T00:00:00Z UTC)**. The unit of time is also known as the **precision**.

Reply:
Counts time since **Unix Epoch (1970-01-01T00:00:00Z UTC)** in specified units (precision).